### DISCUSSION TITLE
Problems arising from flow field inversion at specific points

### DISCUSSION THREAD

**Discussion by 15175829310** [mdolab/dafoam#677] [General]
I selected a specific point in the flow field inverted-v2 script, but found it to be ineffective. Could you please help me check if there are any issues with this script? Thank you
[runScript_v2.txt](https://github.com/user-attachments/files/16851002/runScript_v2.txt)
![Snipaste_2024-09-03_23-18-50](https://github.com/user-attachments/assets/cd0dbad4-6d04-4516-b0eb-d72a6fc84a9f)

**Comment by friedenhe** [comment]
Don't use `fieldInversion` for `type`. Use `variance` instead. Check this example: https://github.com/DAFoam/tutorials/blob/fbc6dcbcbfb006bd21e5e0096e78e66626095194/Ramp/steady/train/runScript_FI.py#L72

**Reply by 15175829310** [reply]
Thank you very much for your reply.But I encountered other problems, specifically as follows: Why does this message appear, one displaying 381 and the other displaying 78, sometimes giving an error because the latter is 0 dots? May I ask why this is happening
![1](https://github.com/user-attachments/assets/1bae9c3c-0953-4b2a-b642-ac26258ef6a5)
![2](https://github.com/user-attachments/assets/cfba083c-59f6-4233-b0bb-ddffc16b2a62)
[logOpt.txt](https://github.com/user-attachments/files/17039873/logOpt.txt)

**Comment by friedenhe** [comment]
The screenshots you posted are some MPI warnings. Please ignore them. I don't see an error here.
